### ISSUE TITLE
The key `root` provided to the classes property is not implemented in Tooltip.

### ISSUE THREAD

**Issue by ANTON072** [DevExpress/devextreme-reactive#1133]
### Current Behaviour

There are warnings from Tooltip Modules.

I use tooltip like this.

```js
<TableHeaderRow showSortingControls messages={{ sortingHint: 'ソート' }}  />
```
Maybe, It's from here.

![image](https://user-images.githubusercontent.com/1940565/40651438-ae100ea2-6370-11e8-9607-b55f8874ad4c.png)

### Your Environment

devextreme-reactive: 1.3.0-beta.1
react: 16.3.1
browser: Chrome
bootstrap: none
react-bootstrap: none
material-ui: 1.0.0

**Comment by kvet** [comment]
https://github.com/DevExpress/devextreme-reactive/issues/1133

Hi,

This issue has been fixed in [1.3.0.beta.2](https://github.com/DevExpress/devextreme-reactive/blob/master/CHANGELOG.md#130-beta2-2018-05-24).

**Comment by ANTON072** [comment]
@kvet Thank you!
